android_sms_gateway/client.py

```
@@ -519,6 +519,57 @@ def revoke_token(self, jti: str) -> None:
 
         self.http.delete(f"{self.base_url}/auth/token/{jti}", headers=self.headers)
 
+    def list_inbox_messages(
+        self,
+        *,
+        filter: t.Optional[domain.InboxQueryFilter] = None,
```

Ruff (0.15.21)

[error] Function argument `filter` is shadowing a Python builtin

(A002)